ranger/config/rc.conf

```diff
@@ -167,7 +170,7 @@ set draw_progress_bar_in_status_bar true
 set draw_borders none
 
 # Display the directory name in tabs?
-set dirname_in_tabs false
+set dirname_in_tabs true
```

Comment:
Don't feel very strongly about this but, again, you give up some info (path of the current directory) in narrow terminals.

Reply:
For very narrow terminals and long paths, that is true. In most other cases, better usability through exposing at least a hint to each tab's location.